composer.json

```diff
@@ -60,6 +60,9 @@
         }
     },
     "autoload": {
+        "branch-alias": {
+            "dev-master": "10.x-dev"
```

Comment:
In my opinion i prefer to wait a bit for the stable version. Buts looks great change more typed, simple to read.